setup.cfg

```diff
@@ -50,11 +50,14 @@ datalad = datalad
 doc =
     nbsphinx
     packaging
-    pydot >=1.2.3
+    pydot >= 1.2.3
     pydotplus
-    sphinx >=1.5.3
+    sphinx >= 2.1.2
     sphinx-argparse
     sphinx_rtd_theme
+    sphinxcontrib-apidoc ~= 0.3.0
+    sphinxcontrib-napoleon
```

Comment:
should the versions in `requirements.txt` also be pinned here?

Reply:
They should not for link-dependencies (i.e., pointing to a github repo and commit directly). If you did then Pypi will not accept the package :(. This is the case of napoleon and versioning.